src/components/weather/HourlyWeather.tsx

```diff
@@ -34,7 +43,9 @@ export const HourlyWeather = ({
         style={styles.icon}
         resizeMode="contain"
       />
-      <RegularText>{temperature.toFixed(1)}°C</RegularText>
+      <RegularText lightest={isNow && isReduceTransparencyEnabled}>
+        {temperature.toFixed(1)}°C
+      </RegularText>
```

Comment:
the text color needs to be checked when merging with all the customers, because it can be that `lightest` and `primary` are not suitable in some cases.

Reply:
yes, we need to be aware of this in the future when merging things